Aztec/Classes/TextKit/LayoutManager.swift

```diff
@@ -47,11 +47,27 @@ class LayoutManager: NSLayoutManager {
             enumerateLineFragments(forGlyphRange:listGlyphRange) { (rect, usedRect, textContainer, glyphRange, stop) in
                 let lineRect = rect.offsetBy(dx: origin.x, dy: origin.y)
                 let lineRange = self.characterRange(forGlyphRange: glyphRange, actualGlyphRange: nil)
+                guard let textList = textStorage.textListAttribute(atIndex: lineRange.location)
+                    else {
+                        return
+                }
+                let number = textStorage.itemNumber(in: textList, at: lineRange.location)
                 let isStartOfLine = textStorage.isStartOfNewLine(atLocation: lineRange.location)
                 var attributes = textStorage.attributes(at: lineRange.location, effectiveRange: nil)
-                attributes[NSParagraphStyleAttributeName] = NSParagraphStyle.Aztec.defaultParagraphStyle
+                attributes[NSParagraphStyleAttributeName] = ParagraphStyle.default
+                attributes.removeValue(forKey: NSUnderlineStyleAttributeName)
+                attributes.removeValue(forKey: NSStrikethroughStyleAttributeName)
+                attributes.removeValue(forKey: NSLinkAttributeName)
+                if let font = attributes[NSFontAttributeName] as? UIFont {
+                    var traits = font.fontDescriptor.symbolicTraits
+                    traits.remove(.traitBold)
+                    traits.remove(.traitItalic)
+                    let descriptor = font.fontDescriptor.withSymbolicTraits(traits)
+                    let newFont = UIFont(descriptor: descriptor!, size: font.pointSize)
+                    attributes[NSFontAttributeName] = newFont
+                }
```

Comment:
You are right, my intention here was to use for the markers the same font that is used in the line, but without any styling or else you end up with bold, italic, striketrouh markers.
I can refactor this to another method in the ParagraphStyle class. 

Another alternative is to have a static font that it's always used for markers...

What do you think @jleandroperez ?